src/AdminCli/CliCommands/DropPriceCommand.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using AdminCli.HttpAccess;
+using AdminCli.NumberParsing;
+using McMaster.Extensions.CommandLineUtils;
+
+namespace AdminCli.CliCommands;
+
+public sealed class DropPriceCommand : ICliCommand
+{
+    public DropPriceCommand(IHttpService httpService) => HttpService = httpService;
+
+    private IHttpService HttpService { get; }
+
+    public void ConfigureCommand(CommandLineApplication config)
+    {
+        config.Description = "Drops the price of a specified product.";
+        var productIdOption =
+            config.Option("-p|--productId <ProductId>",
```

Comment:
please do not use camel casing in options or generally in CLI applications.  this should be `--product-id`